utils/migration_tools/backup_neptune/README.md

```diff
@@ -0,0 +1,94 @@
+# Backup run metadata from Neptune
+
+This script allows you to download run metadata from Neptune to your system. Please note that this script is currently in beta, and we welcome your feedback and contributions to help improve it.
+
+## Prerequisites
+
+Before using this script, make sure you have the Neptune environment variables set up. For instructions, see the [documentation](https://docs.neptune.ai/setup/setting_credentials/).
+
+## Instructions
+
+To use the script, follow these steps:
+
+1. Run `bulk_download_metadata.py`.
+1. The script will generate run logs in the same folder as `bulk_download_metadata.py`. You can modify this location by editing the `logging.basicConfig()` function.
+1. Enter the download path where you want the metadata to be downloaded.
+1. Indicate if you want remotely tracked artifacts to be downloaded.
+1. Enter the projects you want to download the run metadata from.
```

Comment:
Maybe it's not obvious how to enter the project paths? We could link to this: https://docs.neptune.ai/setup/setting_credentials/#project-name

Reply:
The script will print all accessible projects to the console. The names can be copied from there. 
I'll add this info